cmd/balance-agent/main.go

```diff
@@ -303,6 +304,30 @@ func getApp() *cli.App {
 			Value:  "mainnet",
 			Hidden: true,
 		},
+		&cli.StringFlag{
+			Name:   "grpc-url",
+			Usage:  "GRPC URL",
+			Value:  "",
+			Hidden: true,
+		},
+		&cli.Int64Flag{
+			Name:   "fetch-invoices",
```

Comment:
also ambiguous. It can potentially be fetch from datastore or from LN node. But I don't have better suggestion :D